package.json

```diff
@@ -41,7 +41,7 @@
     "test-api": "mocha -R dot --recursive tests/api",
     "test-api-log": "ENABLE_API_ACTIVITY_LOGS=true mocha -R dot --recursive tests/api",
     "test-disablehttp": "REQUIRE_HTTPS=true mocha -R dot --recursive tests/disableHttp",
-    "test-enforced": "REQUIRE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken",
+    "test-enforced": "REQUIRE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken tests/api/v1/profiles",
```

Comment:
So adding this here, we're now executing all the tests under tests/api/v1/profiles _once_ with REQUIRE_ACCESS_TOKEN=true and again with REQUIRE_ACCESS_TOKEN=false? @kriscfoster